3-platform/shopping-cart/domain/src/main/java/com/_6core/platform/shopping/cart/domain/ShoppingCartDomain.java

```diff
@@ -0,0 +1,27 @@
+package com._6core.platform.shopping.cart.domain;
+
+import com._6core.lib.java.domain.model.cart.ShoppingCartV01;
+import java.util.List;
+
+public class ShoppingCartDomain implements ShoppingCartV01 {
```

Comment:
WE dont need this. 
@VictoriaKh-T already created Immutable implementation of ShoppingCartV01
```suggestion
public class ShoppingCartDomain implements ShoppingCartV01 {
```

Reply:
use this https://github.com/sokimaaa/6Core/blob/dev/2-library/domain-model-6core-java/src/main/java/com/_6core/lib/java/domain/model/cart/immutable/ShoppingCartV01Impl.java